lib/puppet_metadata/aio.rb

```diff
@@ -46,6 +46,7 @@ class AIO
         '16.04' => 5..7,
         '18.04' => 5..7,
         '20.04' => 6..7,
+        '22.04' => 6..7,
```

Comment:
In the past Puppet has only added the latest version for a new OS, which makes me suspect they'll only create packages for Puppet 7. However, it's a bit odd where sometimes they do create agent packages but not server packages (like with Debian 11). Should we make it safe until the packages actually exist?
```suggestion
        '22.04' => [], # https://tickets.puppetlabs.com/browse/PA-4233
```

Reply:
I agree with @ekohl

Reply:
With https://github.com/voxpupuli/beaker_puppet_helpers my goal was to make it easy to use OS native packages (again) for when there is no AIO package. While right now Puppet in Ubuntu 22.04 is (sadly) 5.5.22, it was my goal to support Fedora. However, it could still be a decent model to use: initially add an OS, list no packages and use OS native ones. Then when packages appear we can add them here.

Reply:
LGTM, I added the suggested change!